src/tribler/gui/core_manager.py

```diff
@@ -122,8 +125,20 @@ def start_tribler_core(self):
     def on_core_started(self):
         self.core_started = True
         self.core_running = True
+        self.start_port_checker()
         self.events_manager.connect(reschedule_on_err=True)  # retry until REST API is ready
 
+    def start_port_checker(self):
+        core_process_id = self.core_process.processId()
```

Comment:
Can you add a comment for future developers that when using `venv` on Windows, the PID value returned from the `QProcess.processId` is incorrect, as it is the PID of the `venv` wrapper process and not the PID of the Core process itself? Something like:

```suggestion
        # When using venv on Windows, the Core process PID value returned by QProcess is incorrect,
        # as it is the PID of the venv wrapper process and not the PID of the actual Core process.
        # In that case, PortChecker finds nothing, and the default port is used.
        core_process_id = self.core_process.processId()
```